src/components/CreateEventForm/CreateEventForm.tsx

```diff
@@ -86,6 +89,11 @@ export default function CreateEventForm(
     const mm = String(today.getMonth() + 1).padStart(2, '0'); // January is 0!
     const yyyy = today.getFullYear();
     const currentDate = String(`${yyyy}-${mm}-${dd}`);
+    const now = moment().startOf('hour');
```

Comment:
Made the starting time for both inputs begin at the top of the hour so that the initial value cant be something like 14:04.

Reply:
ooo big brain!